packages/twenty-front/src/modules/object-record/record-show/components/RecordShowContainer.tsx

```diff
@@ -246,7 +247,7 @@ export const RecordShowContainer = ({
         <ShowPageLeftContainer forceMobile={isInRightDrawer}>
           {!isMobile && summary}
         </ShowPageLeftContainer>
-        {recordFromStore ? (
+        {recordFromStore && objectNameSingular !== 'activity' ? (
```

Comment:
use CoreObjectNameSingular (which is actually badly name and should be StandardObjectNameSingular...)